src/subscript/ecldiff2roff/ecldiff2roff.py

```diff
@@ -204,7 +207,7 @@ def ecldiff2roff_main(
         filename = outputfilebase + sep + diffpropname + ".roff"
 
         logger.info("Writing to file %s", filename)
-        prop1.to_file(filename, name=diffpropname)
+        diffprop.to_file(filename, name=diffpropname)
```

Comment:
If the file from `prop1.to_file()`  contained more information than the grid dimensions, this will not give the same roff output  (?)

Reply:
A sample test confirms that the output from the two ways of exporting (before and after bugfix) yields the same byte output.